docs/reference/fields.txt

```diff
@@ -397,6 +396,56 @@ matches strings containing "hello" before a newline, besides strings ending in
 This is because the meaning of ``$`` is different between PCRE and Ruby
 regular expressions.
 
+BigDecimal Fields
+-----------------
+
+The ``BigDecimal`` field type is used to store numbers with increased precision.
+
+The ``BigDecimal`` field type stores its values in two different ways under the
+hood, depending on the value of the ``Mongoid.map_big_decimal_to_decimal128``
```

Comment:
`under the hood` --> `in the database`

Reply:
I kind of like it the way it is tbh...

Reply:
"Under the hood":
- Does not clearly indicate under which "hood" -- is it in Ruby's memory or the database?
- Is idiomatic English which does not make sense to non-native speakers (Mongoid is not a car :) )

It's good to think about these points when writing technical documentation.

Reply:
Alright, if you really think so I'll change it

Reply:
done